src/xo-app/vm-import/index.js

```diff
@@ -104,7 +104,10 @@ class VmData extends Component {
       })),
       memory: +refs.memory.value,
       nameLabel: refs.nameLabel.value,
-      networks: map(props.networks, (_, networkId) => refs[`network-${networkId}`].value.id),
+      networks: map(props.networks, (_, networkId) => {
+        const network = refs[`network-${networkId}`].value
+        return network.id ? network.id : network
```

Comment:
I think it's not possible because I think the code path doesn't allow for a null default network.

my thinking is that line 82 would return null  https://github.com/vatesfr/xo-web/blob/6e0dc9aeed1e0955f06136156bd87dcf5ffa5c9d/src/xo-app/vm-import/index.js#L82
and line 86 would crash.